tests/Behat/FeatureContext.php

```diff
@@ -68,12 +68,12 @@ public function assertSelectOptions(string $select, TableNode $options): void {
    * @Then the current workflow state should be :state
    */
   public function assertCurrentWorkflowState(string $state): void {
-    $element = $this->getSession()->getPage()->find('css', 'div#edit-moderation-state-0-current');
+    $element = $this->getSession()->getPage()->find('css', 'div#edit-current');
```

Comment:
This is a bit too generic. The id used is quite dynamic: if any other button in the page is rendered before this one, the id will be different. Let's be more precise so we can be future-proof:
```php
public function assertCurrentWorkflowState(string $state): void {
  // Find the content moderation form.
  $xpath = '//form[@class and contains(concat(" ", normalize-space(@class), " "), " content-moderation-entity-moderation-form ")]'
    // Target the text after the "Moderation state" label.
    . '//label[text()="Moderation state"]/following-sibling::text()[1]';
  $element = $this->getSession()->getPage()->find('xpath', $xpath);
  if (empty($element)) {
    throw new \Exception('The current workflow state field is not present on the page.');
  }
  Assert::assertEquals($state, trim($element->getText()));
}
```